iceoryx_hoofs/source/posix_wrapper/thread.cpp

```diff
@@ -119,9 +116,9 @@ ThreadName_t Thread::getName() const noexcept
         .successReturnValue(0)
         .evaluate()
         .expect("This should never happen! Failed to retrieve the thread name.");
```

Comment:
I would handle the error here in detail but just by providing meaningful error log messages to the user since the underlying functionality of the thread is not compromised.